nebula-query-and-search/main/classes/AggregateQuery.cls-meta.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
-    <apiVersion>58.0</apiVersion>
+    <apiVersion>60.0</apiVersion>
```

Comment:
Since Summer '24 release is now live in all orgs, we could probably bump this to `61.0` now.